File: srinivas/PY_FILES/TASKS/DAY1/t2.py
#!/usr/bin/python
a = [[1,2,3,4,5],{"A":6},(7,8,9)]
c = []
for i in a:
		if type(i) == list:
				c.append(i)
		elif type(i) == dict:
			  c.append(i.values())
		elif type(i) == tuple:
				c.append(i)
for l in c:
		for g in l:
			 print(g)

'''


### dynamic
dynamic = []
x=[1,2,3,4,5]
y={"A":6} 
z=(7,8,9)
dynamic.extend([x,y,z])
print(dynamic)


'''

File: srinivas/PL_FILES/TASKS/First_day/t2.pl
#!/usr/bin/perl
use strict;
use warnings;
use Data::Dumper;
my $a = [[1,2,3,4,5],{"A"=>6},(7,8,9)];
print Dumper($a),"\n";
#my $val = values %{$a->[1]->{'A'}};
#print Dumper($val),"\n";
### dynamic way 
my $cc;
$cc->[0]->[0] = 1;
$cc->[0]->[1] = 2;
$cc->[0]->[2] = 3;
$cc->[0]->[3] = 4;
$cc->[0]->[4] = 5;
$cc->[1]->{"A"} = 6;
$cc->[2][0] = 7;
$cc->[2][1] = 8;
$cc->[2][2] = 9;
print Dumper($cc),"\n";
my @a = @{$cc};
print "@a\n";
